packages/ia-components/sandbox/theatres/theatre-switcher.js

```diff
@@ -39,7 +38,11 @@ export default class TheatreSwitcher extends React.Component {
     let theatreEl = <div>No theatre</div>
     switch (this.state.theatreType) {
       case TheatreSwitcher.THEATRE_TYPES.BOOKREADER:
-        theatreEl = <BookreaderTheatre item={this.props.item} />
```

Comment:
I appreciate this supplemental DOM 👍 .  Let's actually just rip this case out of the switch.